local/php/envs.go

```diff
@@ -29,22 +29,39 @@ import (
 	"github.com/symfony-cli/symfony-cli/envs"
 )
 
-func (p *Server) generateEnv(req *http.Request) map[string]string {
-	scriptName := p.passthru
-	https := ""
-	if req.TLS != nil {
-		https = "On"
-	}
-
-	pathInfo := req.URL.Path
+func (p *Server) resolveScriptName(pathInfo string) (string, string) {
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
-			scriptName = file
-			pathInfo = pathInfo[pos+4:]
+			return file, pathInfo[pos+4:]
+		}
+	}
+
+	if len(pathInfo) > 1 {
+		paths := strings.Split(strings.Trim(pathInfo, "/"), "/")
+		for n := len(paths); n > 0; n-- {
+			if paths[n-1] == "" {
```

Comment:
We use `paths[n-1]` several times here so it's probably a good idea to introduce a variable to get better semantic